sympy/printing/tests/test_jax.py

```diff
@@ -17,7 +17,7 @@
 from sympy.tensor.array.expressions.array_expressions import ArrayTensorProduct, ArrayAdd, \
     PermuteDims, ArrayDiagonal
 from sympy.printing.numpy import JaxPrinter, _jax_known_constants, _jax_known_functions
-from sympy.tensor.array.expressions.conv_matrix_to_array import convert_matrix_to_array
+from sympy.tensor.array.expressions.from_matrix_to_array import convert_matrix_to_array
```

Comment:
I happened to notice this when running the tests. In CI jax isn't tested.

Reply:
Jax was added to the optional dependencies build at https://github.com/sympy/sympy/pull/23627/files. Was it removed at some point?

Reply:
Maybe the warnings are only seen under pytest...

Reply:
Oh I didn't realize this was deprecated. My guess is it gets imported before the warnings filter is enabled. You have to be careful because import deprecations can only be run once, even if the warnings filter is changed. I just checked the CI and it does look like the jax tests are running there.